pilot/pkg/xds/sds.go

```diff
@@ -148,7 +148,10 @@ func (s *SecretGen) Generate(proxy *model.Proxy, w *model.WatchedResource, req *
 			results = append(results, res)
 		}
 	}
-	return results, model.XdsLogDetails{AdditionalInfo: fmt.Sprintf("cached:%v/%v", cached, cached+regenerated)}, nil
+	return results, model.XdsLogDetails{
+		Incremental:    updatedSecrets != nil,
```

Comment:
Is this fixing a bug?

Reply:
Yes but just one impacting logging so not critical at all

Reply:
Oof missed that, sorry for the noise :)